scripts/Muon/GUI/Common/plot_widget/plot_widget_model.py

```diff
@@ -82,3 +125,43 @@ def get_time_workspaces_to_plot(self, current_group_pair, is_raw, plot_type):
             return workspace_list
         except AttributeError:
             return []
+
+    def get_fit_workspace_and_indices(self, fit):
```

Comment:
A minor point but if we have functions like this which don't use self we can make them static. Or even a free function rather than a method. I think this is a nice thing to do because if the method doesn't rely on the state of the object (which is a really good thing) this can be useful to flag for people in the future. Both so they know they can use the method without worrying about state and so they are more cautious of adding state into the method.

Reply:
Yeah good point, I've now converted these and other methods to be static.